### PR TITLE
Lab 3

### PR DESCRIPTION
**Pull request by tprewitt425**

### CODE REVIEW

**Comment by Seven-13** [comment]
Your lab looks really excellent. The 3D plot makes everything look fancy. I think your explanations are detailed and your plots explanatory. The only thing is that I might add some description of the plots in Part 1.A.1 just to mention why the plots are used. Your description of the bounds for the standard normal deviation in Part 1.A.3 is also a bit hard to follow, but it's okay (I can read it and it makes sense, it just seems a bit roundabout). Good job!

**Comment by Pandadrake** [comment]
The report is very well done and thought out. Explanations were very clear and concise. Would be nice to make sure to introduce information such as the probability of sigma five, was confused what number that was for a second.
